kumascript/macros/DeprecatedBadge.ejs

```diff
@@ -10,6 +10,6 @@ var titleAttrValue = mdn.localString({
     "ja"   : "これは非推奨の API です。まだ動作しているかもしれませんが、もう使用するべきではありません。"
 });
 %>
-<svg class="icon icon-deprecated" tabindex="0">
-    <use xlink:href="/assets/badges.svg#icon-deprecated"></use>
-</svg>
+<abbr class="only-icon icon icon-deprecated" title="Deprecated">
+    <span>Deprecated</span>
```

Comment:
Instead of using the `only-icon` CSS class from the browser-compatibility-table module outside of its "scope", let's use `visually-hidden` here on the `<span>` instead.

```suggestion
<abbr class="icon icon-deprecated" title="Deprecated">
    <span class="visually-hidden">Deprecated</span>
```